RULES

```diff
@@ -7,23 +7,57 @@ include $(srcdir)/SRCFILES
 include $(srcdir)/EXTRAFILES
 include $(srcdir)/MISCFILES
 
+#
+# mapping of CPU names to CFLAGS
+#
+MODEL_000 = -m68000
+MODEL_020 = -m68020
+MODEL_02060 = -m68020-60
+MODEL_030 = -m68030
+MODEL_040 = -m68040
+MODEL_060 = -m68060
+MODEL_v4e = -mcpu=5475
+
+#
+# mapping of TARGET names to CPU
+#
+CPU_deb = 02060
```

Comment:
Wouldn't make sense to make this more explicit and map them to the MODEL names?

Reply:
Currently that does not work. CPU names are used in the libkern subdirectory (and a few other places), whereas most of the other Makefiles use kernel target names like ara, hat etc. So unfortunately that has to be mapped twice.

Reply:
Maybe it's worth mentioning this directly in the Makefile then.